src/app/webbrowser/searchengines/ecosia.xml

```diff
@@ -0,0 +1,6 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+ <ShortName>Ecosia</ShortName>
+ <Description>Ecosia, while planting trees</Description>
+  <Url type="text/html" template="https://www.ecosia.org/search?q={searchTerms}"/>
+  <Url type="application/x-suggestions+json" template="https://www.ecosia.org/osjson.aspx?query={searchTerms}"/>
```

Comment:
Morph doesn't appear to be able to get suggestions from this URL. By trying the search on desktop, I found that `https://ac.ecosia.org/?q={searchTerms}` is appropriate for this.

Reply:
Hmm, that doesn't seem to do it either. Wonder why it worked before. I'll get in touch with them.